utility/block.go

```diff
@@ -64,7 +63,7 @@ func (u *UtilityContext) CreateAndApplyProposalBlock(proposer []byte, maxTransac
 			continue
 		}
 		if err := u.Context.IndexTransaction(txResult); err != nil {
-			log.Fatalf("TODO(#327): We can apply the transaction but not index it. Crash the process for now: %v\n", err)
+			u.logger.Fatal().Err(err).Msg("TODO(#327): We can apply the transaction but not index it. Crash the process for now")
```

Comment:
Will we add a `u.logger.TODO` in a separate commit?

Reply:
Probably we should create another ticket for that :)

Reply:
Issue created: https://github.com/pokt-network/pocket/issues/478